Final Project/check.py

```diff
@@ -1,12 +1,16 @@
+import fastai as fastai
```

Comment:
may i know what these two header files are

Reply:
the other one being spark

Reply:
This is the library that I tried to use for hide progress bar in console, You can remove it.

Reply:
Could you remove both the libraries as it is not required

Reply:
Done!

Reply:
I dont think you have commited it to your repo, i cannot see any change here